life_expectancy/cleaning.py

```diff
@@ -0,0 +1,77 @@
+import pandas as pd
+import argparse
+from pathlib import Path
+
+def load_data() -> pd.DataFrame:
+    """ reads life expectancy data into a DataFrame"""
+    path = Path().cwd() / 'life_expectancy' / 'data' / 'eu_life_expectancy_raw.tsv'
```

Comment:
```suggestion
    path = './data/eu_life_expectancy_raw.tsv'
```
We should always use relative paths (hardcoded paths are a pain when sharing code with other developers) and even the path could be an optional function variable where you could have a global variable with the prefix path in case you need it.

Suggestions:
```
BASE_DIR = Path().cwd() / 'life_expectancy' / 'data'

def load_data(path: Optional[Path] = None) -> pd.DataFrame:
    if path is None:
        path = BASE_DIR / 'eu_life_expectancy_raw.tsv'
    return pd.read_csv(path, sep='\t')
```

Reply:
I changed this according to your suggestion.